src/test/e2e-emulated/builtins/require.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { toNano } from "@ton/core";
+import type { SandboxContract, TreasuryContract } from "@ton/sandbox";
+import { Blockchain } from "@ton/sandbox";
+import {
+    Test,
+    GLOBAL_ERROR_VOTING_ENDED,
+    Test_errors_backward,
+} from "./output/require_Test";
+import "@ton/test-utils";
+
+describe("require", () => {
+    let blockchain: Blockchain;
+    let treasury: SandboxContract<TreasuryContract>;
+    let contract: SandboxContract<Test>;
+
+    beforeEach(async () => {
+        blockchain = await Blockchain.create();
+        blockchain.verbosity.print = false;
+        treasury = await blockchain.treasury("treasury");
+        contract = blockchain.openContract(await Test.fromInit());
```

Comment:
Let's reuse the same setup as for `dump`

Reply:
https://github.com/tact-lang/tact/blob/main/src/test/e2e-emulated/debug/dump.spec.ts